src/main/java/com/coherent/training/selenium/stanila/pages/LoginPOM.java

```diff
@@ -18,32 +19,49 @@ public class LoginPOM extends BaseClass{
     private WebElement PASSWORD_INPUT;
     @FindBy(xpath = "//button[@id='passp:sign-in']")
     private WebElement LOGIN_PASSWORD;
-    private Waits waits;
+
+    @FindBy(xpath = "//div[@data-key='view=react-main-buttons']//div//a[@role='button']")
+    private WebElement COMPOSE;
+    public Waits waits;
 
     public LoginPOM() {
         PageFactory.initElements(driver,this);
     }
 
+
     @SneakyThrows
     public void login(String userName, String password){
         waits = new Waits();
+        driver.get("https://mail.yandex.com/");
+        //driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(30));
 
         LOGIN_CENTRAL_PAGE.click();
-        //Thread.sleep(4000);
-        waits.explicitWait(driver, USERNAME_INPUT,15);
+        //Thread.sleep(2000);
+        //waits.waitToBeClickable(driver,USERNAME_INPUT,10);
+        waits.fluentWait(driver,USERNAME_INPUT);
 
         USERNAME_INPUT.sendKeys(userName);
-        waits.explicitWait(driver, LOGIN_USERNAME,15);
+        //Thread.sleep(2000);
+        //waits.waitToBeClickable(driver,LOGIN_USERNAME,10);
+        waits.fluentWait(driver,LOGIN_USERNAME);
```

Comment:
this expression does not make any sense
you can instantiate FluentWait and apply .until with any custom condition you want
example
```
    public static <T> Wait<T> getLongWait(T entry) {
        return WaitUtils.getFluentWait(entry,
                Duration.ofMinutes(FLUENT_WAIT_TIMEOUT_10_MIN),
                Duration.ofMinutes(FLUENT_WAIT_POLLING_EVERY_TIMEOUT_1_MIN),
                Collections.singletonList(NoSuchElementException.class)
        );
    }
```

and 
```
 return WaitUtils.getLongWait(this).until(client -> {
            List<Results> results = client.getResults(url);
            if (results.stream().anyMatch(e -> e.isPresent()) {
                return results;
            } else {
                throw new NoSuchElementException(e + " is not presented in the response ");
            }
        });
```